ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -8,7 +8,7 @@ import java.time.Instant
 import java.util.Date
 
 import anorm.SqlParser.{array, binaryStream, bool, byteArray, date, flatten, get, int, long, str}
-import anorm.{Macro, Row, RowParser, SQL, SimpleSql, SqlParser, SqlStringInterpolation, ~}
+import anorm.{Macro, RowParser, SQL, SqlParser, SqlStringInterpolation, ~}
```

Comment:
just a note: I would have liked better not to review intermittent changes, and review the final version with anorm here instead (which now lives in another PR)

Reply:
I have merged the anorm change into this PR